news/_posts/2015-05-05-release-notes-2.12.0-M1.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: news
+post-type: announcement
+permalink: /news/2.12.0-M1
+title: "Scala 2.12.0-M1 is now available!"
+---
+We are very pleased to announce the release of Scala 2.12.0-M1!
+
+<!-- To be added:
+* Get started with the [Hello Scala 2.11 template](https://typesafe.com/activator/template/hello-scala-2_11) in [Typesafe Activator](https://typesafe.com/platform/getstarted)
+-->
+
+* Download a distribution from [scala-lang.org](http://scala-lang.org/download/2.12.0-M1.html)
+* Obtain it via [Maven Central](http://search.maven.org/?search%7Cga%7C1%7Cg%3A%22org.scala-lang%22%20AND%20v%3A%222.11.0%22#search%7Cga%7C1%7Cg%3Aorg.scala-lang%20AND%20v%3A2.12.0-M1)
+
+Code that compiles on 2.11.x without deprecation warnings should compile on 2.12.x (we do not guarantee this for experimental APIs, such as reflection).
+If not, [please file an issue](https://issues.scala-lang.org).
+
+We are working with the community to ensure availability of the core projects of the Scala 2.12 eco-system.
+This release is *not* binary compatible with the 2.11.x series, to allow us to keep improving the Scala standard library.
+
+The Scala 2.12 series targets Java 8.
+Programs written in Scala 2.12, including the Scala 2.12 compiler, can only be executed on Java 8 or newer.
+Note that the current milestone release (2.12.0-M1) still targets Java 6.
+
+
+### New Features in the 2.12 Series
+
+Scala 2.12 contains all of the bug fixes and improvements made in the 2.11 series.
+
+At the current stage in the milestone cycle, Scala 2.12 is still very similar to Scala 2.11.
+
+The following changes are planned for Scala 2.12:
+  * Java 8 style closures.
```

Comment:
hmm... i pushed a new commit that fixed it when running locally (`jekyll serve`), but it seems that didn't work: http://scala-lang.org/news/2.12.0-M1